app/live-steam-banner/live-stream.tsx

```diff
@@ -104,20 +112,38 @@ export function LiveStream({ event, children }: LiveStreamProps) {
 
   return (
     <>
-      {(showBanner || params?.get("liveBanner")) && (
+      {showBanner ? (
```

Comment:
just wondering, why was the param fetch here removed? it's used to test the live stream banner when the event is coming up

Reply:
As per our conversation, we are maintaining the logic for fetching parameters on the client side. Whenever there are parameters, they will be loaded upon request.